recipes/libbpf/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LibbpfConan(ConanFile):
+    name = "libbpf"
+    description = "eBPF helper library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libbpf/libbpf"
+    license = "LGPL-2.1", "BSD-2-Clause"
+    topics = ("conan", "bpf", "ebpf", "libbpf", "berkeley-packet-filter")
```

Comment:
```suggestion
    topics = ("bpf", "ebpf", "libbpf", "berkeley-packet-filter")
```

Reply:
I just followed the majority of recipes, who add `conan` as a topic. Are there plans to omit that for future recipes?

Reply:
Yes: [H064](https://github.com/conan-io/hooks/blob/5a4810657b358591c5805469f0dd2b78715efc4e/hooks/conan-center.py#L686-L694) checks it.
It's currently a warning.
It looks like it is not (yet) documented in the docs.

Reply:
We have been removing it as PRs get opened =)